OutOfSchool/OutOfSchool.RazorTemplatesData/Services/RazorTemplates.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace OutOfSchool.RazorTemplatesData.Services
+{
+    public static class RazorTemplates
+    {
+        /// <summary>
+        /// Base path to the html email templates
+        /// </summary>
+        private static string MainPath => "/Views/Emails/";
```

Comment:
Consider if `MainPath` should be renamed to something like `MainEmailPath`, Same for `MainPathPlainText`

Reply:
Renamed it, but I'm not sure it's critical. I think it is clear from the context what the main path is.